proto/internal/temporal/server/api/persistence/v1/cluster_metadata.proto

```diff
@@ -39,6 +39,7 @@ message ClusterMetadata {
     bool is_global_namespace_enabled = 9;
     bool is_connection_enabled = 10;
     bool use_cluster_id_membership = 11;
+    map<string,string> tags = 12;
```

Comment:
do we need the same thing for static config?

Reply:
Yes. Will change it in a separate PR.